src/ResourceManager/ServiceBus/Commands.ServiceBus/Utilities/ServiceBusClient.cs

```diff
@@ -270,10 +269,37 @@ public PSQueueAttributes UpdateQueue(string resourceGroupName, string namespaceN
             return new PSQueueAttributes(response);
         }
 
-        public IEnumerable<PSQueueAttributes> ListQueues(string resourceGroupName, string namespaceName)
+        public IEnumerable<PSQueueAttributes> ListQueues(string resourceGroupName, string namespaceName, int? maxCount = null)
         {
-            Rest.Azure.IPage<SBQueue> response = Client.Queues.ListByNamespace(resourceGroupName, namespaceName);
-            IEnumerable<PSQueueAttributes> resourceList = response.Select(resource => new PSQueueAttributes(resource));
+            //Rest.Azure.IPage<SBQueue> response = Client.Queues.ListByNamespace(resourceGroupName, namespaceName);
```

Comment:
@v-Ajnava please remove this commented-out code

Reply:
Resolved